### DISCUSSION TITLE
How will the iBax airdrop be distributed?

### DISCUSSION THREAD

**Discussion by Kendimyazdim** [IBAX-io/go-ibax#248] [Rewards]
hello 
ibax airdrop distribution, will everyone get rights or how much priority will the point system be given to us?

when will the distribution be or what exchange is expected to be listed

**Comment by Emirxt** [comment]
1 Şubattma dağıtımlar başlayacaktır. Puanların karşılığı neyse o kadar verilir daha bilinmiyor ama arkadaşlarınız davet ederek daha çok IBXC kazanabilirsiniz.

**Comment by Emirxt** [comment]
Distributions will begin on February 1st. Points are given as they are, but it is unknown, but you can earn more IBXC by inviting your friends.
